### PR TITLE
Clean up naming

### PR DESCRIPTION
**Pull request by dandydavey**
Clean up naming, get everything building for Vercel

cc: @nichochar

### CODE REVIEW

File: .env.local.example [hunk 1]
```diff
@@ -1,4 +1,3 @@
-  NEXT_PUBLIC_PRIVY_KMS_HOST=https://api.dev.privy.io/v0
-  NEXT_PUBLIC_PRIVY_API_HOST=https://api.dev.privy.io/v0
-  NEXT_PUBLIC_PRIVY_API_KEY=<get public key from https://console.privy.io>
-  PRIVY_API_SECRET=<get private key from https://console.privy.io>
+NEXT_PUBLIC_PRIVY_API_HOST="https://api.privy.io/v0"
```
Comment (nichochar) [inline .env.local.example:1]: Actually, can we remove these? Might as well hard code them, to confuse customers less

File: components/Session.tsx [hunk 2]
```diff
@@ -53,13 +49,17 @@ class Session {
   privy: PrivyClient;
 
   constructor(provider: any) {
-    this.siwe = new SiweSession(NEXT_PUBLIC_PRIVY_API_KEY, provider, {
-      baseURL: NEXT_PUBLIC_PRIVY_API_HOST,
+    console.log("Setting up session: ");
```
Comment (nichochar) [inline components/Session.tsx:52]: Can we remove debug log?

Comment (dandydavey) [reply components/Session.tsx:52]: oops! Totally forgot I added these, yeah I'll take them out, thanks for the catch!

File: .env.local.example (hunk 1 above)
Comment (dandydavey) [reply .env.local.example:1]: good point, yeah I'll take out api host and kms pointers, leaving just the api key for customers to play around with, if they wanted to use this demo as a starting point.

**Pull request merged**